test/removeNPMAbsolutePaths.js

```diff
@@ -406,5 +428,6 @@ describe('removeNPMAbsolutePaths.js', function () {
           expect(writeFile).to.have.been.calledOnce.and.calledWith(filePath);
         });
     });
+
```

Comment:
I think that this empty line is failing the build because of linting error. Please fix it :)